pkg/kubelet/dockertools/exec.go

```diff
@@ -45,14 +49,21 @@ func (*NsenterExecHandler) ExecInContainer(client DockerInterface, container *do
 		return fmt.Errorf("exec unavailable - unable to locate nsenter")
 	}
 
+	cgexec, err := exec.LookPath("cgexec")
+	if err != nil {
+		return fmt.Errorf("exec unavailable - unable to locate cgexec")
```

Comment:
Should this be a hard requirement? Or should we fall back to the old behavior if it's missing?